python/nano/src/bigdl/nano/diffusion/diffusers/pipelines/stable_diffusion.py

```diff
@@ -0,0 +1,237 @@
+    k_dpmpp_2=diffusers.DPMSolverMultistepScheduler,
+    k_euler=diffusers.EulerDiscreteScheduler,
+    k_euler_a=diffusers.EulerAncestralDiscreteScheduler,
+    k_heun=diffusers.HeunDiscreteScheduler,
+    k_lms=diffusers.LMSDiscreteScheduler,
+    plms=diffusers.PNDMScheduler,
+)
+default_scheduler = 'k_lms'
+
+
+def _preload_ov(pretrained_model_name_or_path, device, precision):
+    from bigdl.nano.diffusion.diffusers.modules import NanoUNet
+    unet = load_optimized_unet(
+        cache_dir=os.path.join(pretrained_model_name_or_path, 'unet'),
+        device=device, precision=precision)
+    wrapped_unet = NanoUNet(unet)
+    vae_decoder = load_optimized_vae_decoder(
+        cache_dir=os.path.join(pretrained_model_name_or_path, 'vae'),
+        device=device, precision=precision)
+    return wrapped_unet, vae_decoder
+
+
+def _postload_ov(pipe, unet, value_decoder):
+    # load unet and vae
+    pipe.unet = unet
+    pipe.vae.decoder = value_decoder
+
+
+def _load_ipex(pipe, device, precision):
+    if precision == 'float32':
+        vae = InferenceOptimizer.trace(
+            pipe.vae, device=device,
+            use_ipex=True, channels_last=True)
+        text_encoder = InferenceOptimizer.trace(
+            pipe.text_encoder, device=device,
+            use_ipex=True, channels_last=True)
+        unet = InferenceOptimizer.trace(
+            pipe.unet, device=device,
+            use_ipex=True, channels_last=True)
+    elif precision == 'bfloat16':
+        vae = InferenceOptimizer.quantize(
+            pipe.vae, device=device, use_ipex=True,
+            precision="bf16")
+        text_encoder = InferenceOptimizer.quantize(
+            pipe.text_encoder, device=device,
+            use_ipex=True, precision="bf16")
+        unet = InferenceOptimizer.quantize(
+            pipe.unet, device=device,
+            use_ipex=True, precision="bf16")
+    elif precision == 'float16':
+        vae = InferenceOptimizer.quantize(
+            pipe.vae, device=device,
+            use_ipex=True, channels_last=True,
+            precision="fp16")
+        text_encoder = InferenceOptimizer.quantize(
+            pipe.text_encoder, device=device,
+            use_ipex=True, channels_last=True,
+            precision="fp16")
+        unet = InferenceOptimizer.quantize(
+            pipe.unet, device=device,
+            use_ipex=True, channels_last=True,
+            precision="fp16")
+    else:
+        raise ValueError(f'Unsupported precision {precision}, \
+                         available options are: {["float32", "bfloat32", "float16"]}')
+    pipe.vae = vae
+    pipe.text_encoder = text_encoder
+    pipe.unet = unet
+
+
+class NanoDiffusionPipeline:
+    """NanoDiffusionPipeline."""
+
+    @staticmethod
+    def from_pretrained(
+            cls: DiffusionPipeline,
+            pretrained_model_name_or_path,
+            scheduler=default_scheduler,
+            device='iGPU',
+            precision='float16',
+            backend='OV',
```

Comment:
1. change all default backend to IPEX.
2. move `backend` parameter to the end of the parameter list, so that user can use unamed args for other params.

We can keep the OV related code there but we don't explicitly showcase `backend=` parameter in examples or README. By default we use IPEX and user doesn't have to set it.